config/localTemplate.go

```diff
@@ -460,6 +460,9 @@ type Local struct {
 	// MaxAPIBoxPerApplication defines the maximum total number of boxes per application that will be returned
 	// in GetApplicationBoxes REST API responses.
 	MaxAPIBoxPerApplication uint64 `version[25]:"100000"`
+
+	// EnableTransactionSimulator enables the simulation API endpoint.
+	EnableTransactionSimulator bool `version[25]:"false"`
 }
```

Comment:
I forget if we've already discussed this (I think we only discussed whether to combine this with dev mode configuration.)

I worry we're going to need fairly detailed configuration here - things like "sure, let their simulations run over their opcode budget, but not TOO much", or access many resources, but not TOO many.

Reply:
That's definitely a good point. I won't attempt to add support for this now, but I'll leave this open as a reminder.